python/rapidsmpf/rapidsmpf/tests/conftest.py

```diff
@@ -33,18 +58,21 @@ def _get_mpi_module_or_skip() -> MPI:
     Raises
     ------
     pytest.skip
-        If MPI support is not available.
+        If MPI support is not available or --disable-mpi is specified
     """
     if "mpi" not in COMMUNICATORS:
         pytest.skip("No MPI support")
 
+    if mpi_disabled:
+        pytest.skip("MPI tests disabled")
+
     from mpi4py import MPI
 
     return MPI
 
 
 @pytest.fixture(scope="session")
-def _mpi_comm() -> Communicator:
+def _mpi_comm(_mpi_disabled: bool) -> Communicator:  # noqa: FBT001
```

Comment:
```suggestion
def _mpi_comm(*, _mpi_disabled: bool) -> Communicator:
```

Reply:
Hmm, I thought this would be a positional argument that's why I didn't do it this way. Now I realized you're right, this could not be a positional argument and must be a kwarg, so this in fact works.